miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -69,7 +69,10 @@ abstract class MiniAppMessageBridge {
     @VisibleForTesting
     internal fun createBridgeExecutor(webViewListener: WebViewListener) = MiniAppBridgeExecutor(webViewListener)
 
-    /** Get provided id of mini app for any purpose. **/
+    /**
+     * Get provided id of mini app for any purpose.
+     * You can also throw an [Exception] from this method to pass an error message to the mini app.
```

Comment:
Is `getUniqueId()` throwing any exception?

Reply:
There is a case where the Host App wants to throw an error to the mini app when they fail to generate the Unique ID. For example, the Unique ID could be based on the User's login and they need to fetch it from an API, so the API request could fail.